src/Sulu/Bundle/AdminBundle/Resources/js/views/Datagrid/Datagrid.js

```diff
@@ -24,6 +25,15 @@ class Datagrid extends React.Component<ViewProps> {
     @observable deleting: boolean = false;
     @observable moving: boolean = false;
     @observable showMoveOverlay: boolean = false;
+    @observable toolbarActions = [];
+
+    @computed get selectionDeleting() {
+        if (!this.datagrid) {
+            return false;
+        }
+
+        return this.datagrid.selectionDeleting;
```

Comment:
I think we should move that boolean flag to the `DatagridStore`. This way you could use it directly from the `DatagridStore` in the `DeleteToolbarAction`, and not every component has to implement that on its own.

Reply:
moved to DatagridStore